tests/ui/question_mark.rs

```diff
@@ -186,6 +186,68 @@ fn f() -> NotOption {
     NotOption::First
 }
 
+#[allow(dead_code)]
+mod issue8288 {
```

Comment:
The mod isn't necessary. I'd only reference the issue if it has non-obvious context.

Reply:
well, I did that because I wanted to suppress dead_code warning for all those functions, I'll think of a better way then :)

Reply:
You can add `#![allow(dead_code)]` at the top of the file.

Reply:
Awesome, I was kinda afraid to do that for some reason

Reply:
Yeah we try to keep allows minimal but allowing dead/unused is generally okay for tests.

Reply:
Awesome, I was overthinking other people on this lint might want it, now I guess those function call in `main()` won't be needed